### PR TITLE
Fix/remove search cmd from url

### PR DESCRIPTION
**Pull request by NoelleDL**
### What

Remove `?service=cmd` from url

### How to review

 Click on feedback link in Beta banner. See that: 

- url does not contain `?service=cmd`
- url populates on feedback page
- disabling JS does not affect the feedback journey

### Who can review

Anyone but me.

### CODE REVIEW

File: assets/templates/partials/banners/search-beta.tmpl [hunk 1]
```diff
@@ -3,7 +3,7 @@
         <div class="col col--md-40 col--lg-44">
             <span class="banner__text-icon {{ if eq .Type "search" }} banner__text-icon--double-line {{ end }}">{{ localise "Beta" .Language 1 }}</span>
             <div class="banner__body">
-                <p>{{ localise "BetaCMDFeedback" .Language 1 | safeHTML }} </p>
+                <p>{{ localise "BetaSearchFeedback" .Language 1 | safeHTML }} </p>
                 {{ if eq .Type "search" }} 
                     <form action="/search" method="get">
                         <input type="hidden" id="q" name="q" value="{{ .Data.Query }}">
```
Comment (janderson2) [inline assets/templates/partials/banners/search-beta.tmpl:9]: I realise this is scope creep so feel free to ignore, but on line 11 below the text `Return to the previous version` should be using a dictionary code.

File: assets/templates/partials/feedback.tmpl [hunk 2]
```diff
@@ -6,10 +6,10 @@
                 Yes
             </a>
             <span>|</span>
-            <a id="feedback-form-no" class="js-toggle" href="/feedback?service=cmd" aria-label="No I didn't find this page useful">
+            <a id="feedback-form-no" class="js-toggle" href="/feedback" aria-label="No I didn't find this page useful">
                 No
```
Comment (janderson2) [inline assets/templates/partials/feedback.tmpl:10]: This file should also be using dictionary tags for the text values and the aria label text.

File: assets/templates/partials/header.tmpl [hunk 3]
```diff
@@ -270,6 +270,6 @@
         </div>
     {{end}}
     {{ if and .BetaBannerEnabled .FeatureFlags.SixteensVersion }}
-        {{ template "partials/banners/cmd-beta" . }}
+        {{ template "partials/banners/search-beta" . }}
```
Comment (mr-nick17) [inline assets/templates/partials/header.tmpl:273]: The `search-beta` template isn't in source control. Do you need to push the file rename?

**Comment by mr-nick17** [comment]
Not so much a code query but more of an implementation detail, is the intention to remove the beta banner from CMD? If so, it might be worth changing the feature flag in `dp-frontend-dataset-controller` first. If not, and the plan is run two banners in tandem (cmd and search) you'll need to implement a flag in the service that propagates down to the renderer

**Review by mr-nick17 — APPROVED** [review]
LGTM

**Pull request merged**